nncf/common/pruning/mask_propagator.py

```diff
@@ -0,0 +1,59 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from nncf.common.graph.graph import NNCFGraph
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.pruning.export_helpers import DefaultMetaOp
+from nncf.common.pruning.utils import PruningOperationsMetatypeRegistry
+
+
+class MaskPropagator:

```

Comment:
I would like to suggest for discussion the following name of the pruning mask propagation algorithm `MaskPropagationAlgorithm` instead of `MaskPropagator`. 
@AlexanderDokuchaev  @evgeniya-egupova @mkaglins @ljaljushkin @vshampor what do you think?

Reply:
Yes, MaskPropagationAlgorithm is better 👍